### ISSUE TITLE
Acoustic similarity: results options

### ISSUE THREAD

**Issue by kchall** [PhonologicalCorpusTools/CorpusTools#394]
Although I'm really glad that PCT now returns all of the pairwise comparisons in a directory or between directories, that can sometimes be overwhelming. Can we make that optional? E.g., have a checkbox for "Return summary result only" or something?

**Comment by mmcauliffe** [comment]
Yeah, I'd actually been thinking about overhauling the results windows for everything to have detailed results and summary results by default, with maybe a switch to disable summarization when it's not relevant.

This would make acoustic similarity and phono search pretty similar, but we could also have details for minimal pairs or neighborhood density, and maybe even general feature summaries vs segment details for things like predictability of distribution environments

**Comment by mmcauliffe** [comment]
I think to do this right, we'd probably need to spend some time on it, so I'm going to move it to 1.2.
